src/main/java/net/sf/jabref/logic/util/strings/StringUtil.java

```diff
@@ -167,7 +167,7 @@ public static String wrap(String in, int wrapAmount) {
 
         String[] lines = in.split("\n");
         StringBuilder result = new StringBuilder();
-        addWrappedLine(result, lines[0], wrapAmount);
+        addWrappedLine(result, lines[0].replaceAll("\\s+$", ""), wrapAmount);
```

Comment:
It is stated a few lines down. The idea is to remove \r if the line breaking is \r\n, but it turned out that it didn't work on the first line, only on the remaining ones.

However, now I realize that if the line-breaking is only \r (MAC, right?) the complete function will fail its purposed. Still, the current version is at least slightly better than the earlier...